### PR TITLE
fix pulumi dot

### PR DESCRIPTION
**Pull request by lionello**
Partial fix for #416

### CODE REVIEW

File: src/pkg/cli/tail.go [hunk 1]
```diff
@@ -193,17 +193,18 @@ func Tail(ctx context.Context, client client.Client, params TailOptions) error {
...
-				time.Sleep(time.Second)
+				pkg.SleepWithContext(ctx, 1*time.Second)
 				serverStream, err = client.Tail(ctx, &defangv1.TailRequest{Service: params.Service, Etag: params.Etag, Since: timestamppb.New(params.Since)})
 				if err != nil {
 					term.Debug(" - Reconnect failed:", err)
 					return err
 				}
 				if !params.Raw {
-					term.Fprint(term.Stderr, term.WarnColor, " ! Reconnected!   \r") // overwritten with logs
+					fmt.Fprintf(term.Stderr, "%*s", spaces, "\r") // clear the "reconnecting" message
```
Comment (edwardrf) [inline src/pkg/cli/tail.go:207]: nice hack

File: src/pkg/term/colorizer.go [hunk 2]
```diff
@@ -135,3 +136,9 @@ func Fatalf(format string, v ...any) {
 	Errorf("Error: "+format, v...)
 	os.Exit(1)
 }
+
+var ansiRegex = regexp.MustCompile("\x1b(?:\\[[=?]?[0-9;]{0,10}[@-~]|].{0,10}?(?:\x1b\\\\|\x07|$)|[@-Z\\\\^_])")
```
Comment (edwardrf) [inline src/pkg/term/colorizer.go:140]: Do we have a link for this Regex or explanation for it?

**Review by edwardrf — APPROVED** [review]
LGTM

File: src/pkg/term/colorizer.go (hunk 2 above)
Comment (lionello) [reply src/pkg/term/colorizer.go:140]: yeah let me look for it.

**Review by edwardrf — APPROVED** [review]
LGTM

**Pull request merged**